packages/core/admin/_internal/node/core/plugins.ts

```diff
@@ -79,14 +110,16 @@ const getEnabledPlugins = async ({
 
   for (const [userPluginName, userPluginConfig] of Object.entries(userPluginsFile)) {
     if (userPluginConfig.enabled && userPluginConfig.resolve) {
+      const sysPath = convertModulePathToSystemPath(userPluginConfig.resolve);
       plugins[userPluginName] = {
         name: userPluginName,
-        isLocal: true,
+        type: 'local',
         /**
          * User plugin paths are resolved from the entry point
          * of the app, because that's how you import them.
          */
-        pathToPlugin: userPluginConfig.resolve,
+        modulePath: path.relative(runtimeDir, convertSystemPathToModulePath(sysPath)),
```

Comment:
shouldn't this be `convertSystemPathToModulePath(path.relative(runtimeDir, sysPath))` ?

Reply:
good catch, i wrapped the wrong function – sometimes i do wish i had windows 🙃